docs/containers/debug-python.md

```diff
@@ -107,7 +107,123 @@ Here is an example of using `dockerServerReadyAction` to launch the browser to o
 
 - `uriFormat`: By default, the Docker extension will open the main page of the browser (however that is determined by the application). If you want the browser to open a specific page like the example above, the `uriFormat` property of the [dockerServerReadyAction](debug-common.md#dockerServerReadyAction-object-properties) object should be set to a format string with two string tokens to indicate the protocol and port substitution.
 
-### How to debug your app with Gunicorn
+## How to enable hot reloading in Django or Flask apps
+
+When you select **Docker: Add Docker Files to Workspace** for Django or Flask, we provide you a Dockerfile and `tasks.json` configured for static deployment. Each time you make changes to your app code, you will need to rebuild and re-run your container. Hot reloading allows you to visualize changes in your app code as your container continues to run. Enable hot reloading with these steps:
+
+### For Django Apps
+
+1. In the Dockerfile, comment out the line that adds app code to the container.
+
+    ``` dockerfile
+    #ADD . /app
+    ```
+
+1. Within the `docker-run` task in the `tasks.json` file, create a new dockerRun attribute with a volumes property. This will create a mapping from the current workspace folder (app code) to the `/app` folder in the container.
+
+    ``` json
+    {
+      "type": "docker-run",
+      "label": "docker-run: debug",
+      "dependsOn": [
+        "docker-build"
+      ],
+      "dockerRun": {
+        "volumes": [
+          {
+            "containerPath": "/app", "localPath": "${workspaceFolder}"
+          }
+        ]
+      },
+      ...
+    }
+    ```
+
+1. Edit the python attribute by **removing** `--noreload` and `--nothreading`.
+
+    ``` json
+    {
+      ...
+      "dockerRun": {
+        "volumes": [
+          {
+            "containerPath": "/app", "localPath": "${workspaceFolder}"
+          }
+        ]
+      },
+      "python": {
+        "args": [
+          "runserver",
+          "0.0.0.0:8000",
+        ],
+      }
+      ...
+    }
+    ```
+
+1. Select the **Docker: Python – Django** launch configuration and hit `kb(workbench.action.debug.start)` to build and run your container.
+1. Modify and save an HTML file.
+1. Refresh the browser and validate changes have been made.
+
+### For Flask Apps
+
+1. In the Dockerfile, comment out the line that adds app code to the container.
+
+    ``` dockerfile
+    #ADD . /app
+    ```
+
+1. Within the `docker-run` task in the `tasks.json` file, create a new dockerRun attribute with a volumes property. This will create a mapping from the current workspace folder (app code) to the `/app` folder in the container.
```

Comment:
If users are using the default scaffold, then there will be an existing `dockerRun` attribute.

Reply:
Changed the wording to "edit the existing dockerRun attribute..."